### PR TITLE
impt auth enhancements

### PR DESCRIPTION
**Pull request by ekaterina-nbl**

### CODE REVIEW

File: CommandsManual.md [hunk 1]
```diff
@@ -254,7 +293,7 @@ impt auth login [--local] [--endpoint <endpoint_url>]
     [--output <mode>] [--help]
 ```
 
-Perform global or local login. Creates a [global](#global-auth-file) or [local](#local-auth-file) auth file.
+Creates the [global](#global-auth-file) or [local](#local-auth-file) auth file.
```
Comment (ppetrosh) [inline CommandsManual.md]: .. a global ... or a local ...

Comment (alexey-nbl) [reply CommandsManual.md]: I do not know how it's correct in English.
As the global auth file can be one and only one.
Please update how you think is right or let's give a chance to Tony to update.

Comment (smittytone) [reply CommandsManual.md]: Use 'the' when there is only one item. Use 'a' when you are talking about one item among many items.

Comment (alexey-nbl) [reply CommandsManual.md]: @smittytone I know this general rule but do not understand how to apply it here :)
Global file can be only one per impt installation. Local files can be many, but only one in a particular directory.
So, what is correct here?
"Creates the global or a local auth file" ?
Thanks.

**Pull request merged**